config/jobs/image-pushing/k8s-staging-node-ipam-controller.yaml

```
@@ -0,0 +1,24 @@
+postsubmits:
+  kubernetes-sigs/node-ipam-controller:
+    - name: post-node-ipam-controller-push-images
+      cluster: k8s-infra-prow-build-trusted
+      annotations:
+        testgrid-dashboards: sig-network-node-ipam-controller, sig-k8s-infra-gcb
+        testgrid-alert-email: antonio.ojea.garcia@gmail.com,neverov.max@gmail.com
+      decorate: true
+      branches:
+        - ^main$
+        - ^release-
+        # Build semver tags, too
+        - ^v(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$
+      spec:
+        serviceAccountName: gcb-builder
+        containers:
+          - image: gcr.io/k8s-staging-test-infra/image-builder:v20230711-e33377c2b4
+            command:
+              - /run.sh
+            args:
+              - --project=k8s-staging-networking
+              - --scratch-bucket=gs://k8s-staging-networking-gcb
+              - --env-passthrough=PULL_BASE_REF
+              - .
```

```suggestion
              - --build-dir=.
              - --with-git-dir
              - .
```

you need to pass the git dir for the script in the makefile to get the right value

https://github.com/kubernetes-sigs/node-ipam-controller/blob/1323bc8a3524b800c5c757d06324a6978e81bf6d/Makefile#L15C1-L16C1

---

Updated, thank you!